starport/errors/errors.go

```diff
@@ -0,0 +1,9 @@
+// Package sperrors holds starport spesific errors.
+package sperrors
+
+import "errors"
+
+var (
+	// ErrOnlyStargateSupported is returned when underlying chain is not a stargate chain.
+	ErrOnlyStargateSupported = errors.New("only stargate chains are supported")
```

Comment:
```suggestion
	ErrOnlyStargateSupported = errors.New("This version of Cosmos SDK is no longer supported.")
```

Reply:
Go errors must not have uppercase or punctuation though I agree with the message content

Reply:
d327fa8635c333d733fe17ddce79bf92a38b9e72